.NET/Microsoft.Recognizer.Text.TimeZone/English/TimeZoneParser.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Text.RegularExpressions;
+using Microsoft.Recognizers.Definitions.English;
+
+namespace Microsoft.Recognizers.Text.TimeZone.English
+{
+    class TimeZoneParser : IParser
+    {
+        public TimeZoneParser()
+        {
+
+        }
+        public int getMinutes(string matched)
+        {
+            int bs = 1;
+            if (matched.StartsWith("+")
+                || matched.StartsWith("-"))
+            {
+                if (matched.StartsWith("-"))
+                    bs = -1;
+
+                matched = matched.Substring(1).Trim();
+            }
+            int h = 0;
```

Comment:
Please give variables better names (ex: hours, minutes) this increases the readability of the code and help with future maintenance.